features/support/helpers.rb

```diff
@@ -149,13 +154,13 @@ def seconds_agnostic_datetime(datetime = Time.now)
     "#{time}:\\d{2}",
     Regexp.escape(zone)
   ] \
-  .join("\\ ")
+    .join("\\ ")
 end
 
 #
 
 def seconds_agnostic_time(time)
   time = time.strftime("%H:%M:%S") if time.is_a?(Time)
-  hour, minutes, _ = time.split(":")
+  hour, minutes, = time.split(":")
```

Comment:
It is a violation of `Style/TrailingUnderscoreVariable`

I guess a "better" way to do this would be

```ruby
hour, minutes = time.split(":")
```